packages/paste-website/src/pages/getting-started/design/index.mdx

```diff
@@ -112,36 +89,76 @@ The main libraries we recommend you have enabled for each file are:
+When you have a theme library linked, you'll have access to:
+- Text styles (font families, font weights, and font sizes)
+- Colour styles (fill and stroke for background, border, and text colors)
+- Effect styles (shadows, etc.)
 
 ![Menu in Figma when you can navigate through tokens for the Default theme](../../../assets/images/design-guidelines/token-styles.png)
 
-You should be using components from Paste to compose your UIs whenever possible, but the text and layer styles will be helpful when you need to compose something more custom.
+You should be using components from Paste to compose your UIs whenever possible, but the text and layer styles will be helpful when you need to compose something more custom. 
+
 This can be anything from setting a piece of text next to an icon to creating a custom stepper component.
 
-### How we publish library updates
+---
+
+## Using components
+  
+Components are UI elements that can be reused across your designs. They help you create and manage consistent designs across projects.
+
+You can drag and drop components into your project via the Assets panel on the left, and swap to different variants for a given component via the Design panel on the right. 
+
+Any layer with a ⚙️ emoji indicates that there are nested variants available. 
+
+Any layer with a ✏️ emoji indicates an editable layer. 
 
-We release library updates continuously, and we demarcate changes with hints which you can see in the description of each update:
+Check out our [tips and tricks](https://www.figma.com/file/E6KUvMhioUmAgN0nwmReTM/Paste-Components?node-id=284%3A1158) for more helpful hints about using the Paste Components library.
+
+---
+
+## Library updates
+
+We release library updates continuously and use Figma’s version history to keep track.
+
+We demarcate changes with hints which you can see in the description of each update:
 
 - 🚀&nbsp;&nbsp;**New**: [summary of a new thing]
 - ✨&nbsp;&nbsp;**Improved**: [summary of an improved thing]
 - 🐛&nbsp;&nbsp;**Fixed**: [summary of a bug fix]
 - ❗️&nbsp;&nbsp;**Breaking**: [summary of a breaking change]
 
+---
+
+## Fonts
+
+Fonts in Paste are preloaded into Figma so you don't have to worry about downloading or installing them.
+
+### Fira Mono
+
+[Fira Mono](https://fonts.google.com/specimen/Fira+Mono?query=fira+mono) is a free Google font and our monospace typeface for code.
+  
+### Inter
+
+[Inter](https://fonts.google.com/specimen/Inter) is a free Google font and our typeface for UI text.
+
+### SF Pro Text
+
+[SF Pro Text](https://developer.apple.com/fonts/) is the system font for Apple platforms. We represent native web element styles (like option lists in [Select](/components/select/)) in our design assets in macOS styles.
+
+### Whitney
+  
+If you are using the <strong>Console theme</strong> in Paste, you will need to download Whitney, which we use for UI text.
+Reach out to us in `#help-design-system` if you need access to the Whitney font files for your work.
```

Comment:
```suggestion
[Whitney](https://www.twilio.com/brand/elements/typography) is the Twilio brand typeface. Reach out to us in `#help-design-system` if you need access to the Whitney font files for work outside of Figma.
```

Reply:
@katieporter ok to remove mention of the console theme here?

Reply:
yes plz!